tests/foreman/maintain/test_clone.py

```diff
@@ -0,0 +1,163 @@
+"""Tests for satellite-maintain clone
+
+:Requirement: foreman-maintain
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: SatelliteClone
+
+:Assignee: lpramuk
```

Comment:
@lpramuk you know @LadislavVasina1 is making you own this module? :wink:

Reply:
SatelliteClone = lpramuk, 
it could change in future